src/components/CreateShareStructure/ListShareClass.vue

```diff
@@ -1,14 +1,368 @@
+          <td>{{seriesItem.maxNumberOfShares || 'No Maximum'}}</td>
+          <td>{{row.item.parValue ? `$${row.item.parValue}.00` : 'No Par Value'}}</td>
+          <td>{{row.item.currency}}</td>
+          <td>{{seriesItem.hasRightsOrRestrictions ? 'Yes' : 'No'}}</td>
+
+          <!-- Share Series Edit Btn -->
+          <td v-if="!isSummary">
+            <div class="actions">
+              <span class="edit-action">
+                <v-btn small text color="primary"
+                   :id="'Series-' + index + '-change-btn'"
+                   @click="emitShareSeries(row.index, index)"
+                >
+                  <v-icon small>mdi-pencil</v-icon>
+                  <span>Edit</span>
+                </v-btn>
+              </span>
+
+              <!-- Share Series Dropdown Actions -->
+              <span>
+                  <v-menu offset-y>
+                    <template v-slot:activator="{ on }">
+                      <v-btn text small color="primary"
+                        class="actions__more-actions__btn" v-on="on"
+                      >
+                        <v-icon>mdi-menu-down</v-icon>
+                      </v-btn>
+                    </template>
+                    <v-list class="more-actions">
+                      <v-list-item
+                        class="actions-dropdown_item"
+                        :class="{ 'item-disabled': isMoveDisabled(row.index, 'up', index) }"
+                        @click="moveIndex(row.index, 'up', index)"
+                        :disabled="isMoveDisabled(row.index, 'up', index)"
+                      >
+                        <v-list-item-subtitle><v-icon>mdi-arrow-up</v-icon> Move Up</v-list-item-subtitle>
+                      </v-list-item>
+                      <v-list-item
+                        class="actions-dropdown_item"
+                        :class="{ 'item-disabled': isMoveDisabled(row.index, 'down', index) }"
+                        @click="moveIndex(row.index, 'down', index)"
+                        :disabled="isMoveDisabled(row.index, 'down', index)"
+                      >
+                        <v-list-item-subtitle><v-icon>mdi-arrow-down</v-icon> Move Down</v-list-item-subtitle>
+                      </v-list-item>
+                      <v-list-item class="actions-dropdown_item" @click="emitRemoveSeries(row.index, index)">
+                        <v-list-item-subtitle><v-icon>mdi-delete</v-icon> Remove</v-list-item-subtitle>
+                      </v-list-item>
+                    </v-list>
+                  </v-menu>
+                </span>
+            </div>
+          </td>
+        </tr>
+      </template>
+    </v-data-table>
+  </v-card>
 </template>
 
 <script lang="ts">
 // Libraries
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Prop, Vue, Emit } from 'vue-property-decorator'
+import 'array.prototype.move'
+
+// Components
+
+// Dialogs
+
+// Mixins
+
+// Interfaces
```

Comment:
Remove the ones you're not using (line 169-175).

Reply:
Done